File: 3rd Semester/Advanced Programming Methods/exams session/flights/java/domain/Entity.java
package ubb.scs.map.zboruri.domain;

public class Entity<ID> {
    private ID identityKey;

    public ID getID() {
        return identityKey;
    }

    public void setID(ID identityKey) {
        this.identityKey = identityKey;
    }
}

File: 4th Semester/Design and Programming Environments/labs/4th lab/C#/BasketModel/Entity.cs
using System;
namespace BasketModel
{
    public class Entity<ID>
    {
        private ID identitykey;
        public ID get_identitykey() {
            return identitykey;
        }
        public void set_identitykey(ID identitykey) {
            this.identitykey = identitykey;
        }
    }
}
